core/engine/src/builtins/temporal/plain_year_month/mod.rs

```diff
@@ -410,25 +362,35 @@ impl PlainYearMonth {
         add_or_subtract_duration(false, this, duration_like, &options, context)
     }
 
+    /// 9.3.16 `Temporal.PlainYearMonth.prototype.until ( other [ , options ] )`
     fn until(_this: &JsValue, _: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
-        Err(JsNativeError::typ()
+        Err(JsNativeError::error()
             .with_message("not yet implemented.")
             .into())
     }
 
+    /// 9.3.17 `Temporal.PlainYearMonth.prototype.since ( other [ , options ] )`
     fn since(_this: &JsValue, _: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
-        Err(JsNativeError::typ()
+        Err(JsNativeError::error()
             .with_message("not yet implemented.")
             .into())
```

Comment:
Good to know we're returning `Error` instead of `TypeError` for unimplemented features to avoid false positives.